<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>621</width>
       <height>387</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone 1 - Page 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>191</width>
        <height>141</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 157, 157);</string>
      </property>
      <property name="title">
       <string>State / City</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Academy Engraved LET</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 248, 248);
color: rgb(27, 28, 28);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cities</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Academy Engraved LET</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 248, 248);
color: rgb(31, 31, 31);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Zipcodes</string>
       </property>
      </widget>
      <widget class="QListWidget" name="zipcodeList">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Academy Engraved LET</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 248, 248);
color: rgb(31, 31, 31);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="businessGroup2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>421</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 157, 157);</string>
      </property>
      <property name="title">
       <string>Businesses</string>
      </property>
      <widget class="QTableWidget" name="businessTable2">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>20</y>
         <width>431</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Academy Engraved LET</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 248, 248);
color: rgb(43, 44, 44);</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="ZipCodeInfoGroup">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>201</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 157, 157);</string>
      </property>
      <property name="title">
       <string>ZipCodeInfo</string>
      </property>
      <widget class="QTableWidget" name="zipInfoTable">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Academy Engraved LET</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 248, 248);
color: rgb(43, 44, 44);</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="TopCatgoriesGroup">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>0</y>
        <width>211</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 157, 157);</string>
      </property>
      <property name="title">
       <string>TopCategories</string>
      </property>
      <widget class="QTableWidget" name="topCategoriesTable">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>211</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Academy Engraved LET</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 248, 248);
color: rgb(43, 44, 44);</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="successfulGroup">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>260</y>
        <width>301</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 157, 157);</string>
      </property>
      <property name="title">
       <string>Successful Businesses</string>
      </property>
      <widget class="QTableWidget" name="successfulTable">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>301</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Academy Engraved LET</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 248, 248);
color: rgb(43, 44, 44);</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="popularGroup">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>260</y>
        <width>301</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 157, 157);</string>
      </property>
      <property name="title">
       <string>Popular Businesses</string>
      </property>
      <widget class="QTableWidget" name="popularBusinessTable">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>301</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Academy Engraved LET</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 248, 248);
color: rgb(43, 44, 44);</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
     <widget class="QGroupBox" name="FilterCategoriesGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 157, 157);</string>
      </property>
      <property name="title">
       <string>Filter By Categories</string>
      </property>
      <widget class="QTableWidget" name="filterCategoriesTable">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Academy Engraved LET</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(244, 248, 248);
color: rgb(43, 44, 44);</string>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
